src/compiler/diagnosticMessages.json

```diff
@@ -1809,6 +1809,10 @@
         "category": "Error",
         "code": 1538
     },
+    "A `bigint` literal may not be used as a property name.": {

```

Comment:
It's throwing me off that half the messages in the output use a different quote style. Let's stay consistent with the majority of uses.

Reply:
Let's also stay consistent with "cannot" instead of "may not".